### PR TITLE
Updated with new gotham headers and brand colors (yay!)

### PR DESCRIPTION
**Pull request by nmcclay**

### CODE REVIEW

**Comment by nmcclay** [comment]
screenshot 2015-09-04 12 39 57

**Comment by nmcclay** [comment]
screenshot 2015-09-04 12 40 34

File: app/views/forever_style_guide/sections/visual_elements/typography/_headings.erb [hunk 1]
```diff
@@ -7,37 +7,37 @@
...
       <td class="type-info font-size-700">
-        <label>Light</label>
+        <label></label>
       </td>
     </tr>
     <tr>
       <td><h2>h2. Forever heading</h2></td>
       <td class="type-info font-size-600">
-        <label>Light</label>
+        <label></label>
```
Comment (schweitzdesign) [inline app/views/forever_style_guide/sections/visual_elements/typography/_headings.erb]: Remove empty elements

File: app/assets/stylesheets/forever_style_guide/globals/_colors.scss [hunk 2]
```diff
@@ -4,15 +4,23 @@
 ---------------------------------------------------------------------------- */
 
 // Core color declarations, this list is intended to be a limited set of unique branded colors
-$color-forever_green:   colorify('green', #7ac143) !default;
-$color-forever_blue:    colorify('blue', #00a4e4) !default;
-$color-forever_purple:  colorify('purple', #731472) !default;
+$color-forever_green:   colorify('green', #7dc142) !default;
+$color-forever_spare:   colorify('spare', #7ec24d) !default;
```
Comment (schweitzdesign) [inline app/assets/stylesheets/forever_style_guide/globals/_colors.scss]: what is forever_spare?

**Comment by schweitzdesign** [comment]
Swatches look awesome. Looks like the headings are Helvetica though?

File: app/assets/stylesheets/forever_style_guide/globals/_colors.scss (hunk 2 above)
Comment (nmcclay) [inline app/assets/stylesheets/forever_style_guide/globals/_colors.scss]: we have one spare color in that spectrum for future use, I didn't know what to call it so 'spare' seemed appropriate for now.

**Comment by schweitzdesign** [comment]
This looks really good. The only item out there that looks a bit unexpected w/ the Gotham merge is that Panel Titles are marked up w/ h3 and, in turn, Gotham, while the content and footer is in Proxima.  Just making a note here that if we implement a strict bootstrap panel somewhere we should make sure there's ample differentiation.

screen shot 2015-09-04 at 3 06 38 pm

**Pull request merged**